aria-practices.html

```diff
@@ -3734,6 +3734,390 @@ <h5>Examples</h5>
+            <td>Enables a live region.</td>
+           </tr>
+          <tr>
+            <td><code>aria-atomic</code></td>
+            <td>Indicates what content to announce.</td>
+          </tr>
+          <tr>
+            <td><code>aria-relevant</code></td>
+            <td>Indicates which content changes are relevant.</td>
+          </tr>
+          <tr>
+            <td><code>aria-busy</code></td>
+            <td>Defers content updates. Not specific to live regions. See the <a href="#aria-busy">next section</a>.</td>
+          </tr>
+        </tbody>
+      </table>
+
+      <section id="enable_live_regions_with_aria-live">
+        <h4>Enable Live Regions with <code>aria-live</code></h4>
+
+        <p>
+          The <code>aria-live</code> attribute indicates that an element is a live region.
+          Some roles implicitly set <code>aria-live</code>; this is discussed in a later section.
+        </p>
+
+        <ul>
+          <li><code>assertive</code>: assistive technologies can interrupt the user to provide this information,
+          but will not move the current focus, so the user&#39;s work flow is not interrupted.</li>
+
+          <li><code>polite</code>: assistive technologies will provide this information after the user is done with their current task.</li>
+
+          <li><code>off</code>: assistive technologies will not provide this information unless this region has focus.</li>
+        </ul>
+
+        <p>
+          With the exception of a few roles, the default value of <code>aria-live</code> is <code>off</code>.
+        </p>
+
+        <p>
+          For example, when the user types into a search field, the page could update the page with search results as the user is typing.
+          To inform users of assistive technology that this has happened, a polite live region can be used.
+          When the user is done typing, and the search is complete, the user is informed of how many results were found,
+          without having to move focus away from the search field.
+        </p>
+
+        <pre><code>&lt;form role="search" aria-labelledby="search">
+ &lt;h2 id="search">Search&lt;/h2>
+ &lt;label>Search query: &lt;input type="search" name="q" oninput="updateSearch(event)">&lt;/label>
+ &lt;div id="search-result-status" role="region" aria-live="polite">&lt;/div>
+&lt;/form>
+&lt;script>
+async function updateSearch(event) {
+  const statusElement = document.getElementById(&#39;search-result-status&#39;);
+  const results = await getSearchResults(event.target.value);
+  statusElement.textContent = `${results.length} result(s) found.`;
+  showResults(results);
+}
+&lt;/script>
+</code></pre>
+      </section>
+
+      <section id="indicate_what_content_to_announce_with_aria-atomic">
+        <h4>Indicate What Content to Announce with <code>aria-atomic</code></h4>
+
+        <p>
+          The <code>aria-atomic</code> attribute takes the values "true" and "false". The attribute can also be omitted.
```

Comment:
```suggestion
          The <code>aria-atomic</code> attribute takes the values "true" and "false". The attribute can also be omitted, as discussed below.
```

To keep the first-time reader from thinking that discussion of the default value must be missing.